ArticleTemplates/assets/js/modules/creativeInjector.js

```diff
@@ -18,30 +18,48 @@ function (
         }
     }
 
+    function trackLiveBlogEpic() {
+        // if there is already a data-tracked attribute than we don't need to set up tracking again
+        var liveBlogEpicContainer = document.querySelectorAll('.contributions-epic:not([data-tracked])');
+        var liveBlogEpicContainerId;
+
+        if (liveBlogEpicContainer) {
```

Comment:
we should replace this if block with a for loop as liveBlogEpicContainers will be a list potentially containing multiple epic HTML elements, we'll need to set up tracking on each element in this node list. eg.

```
var i;
var liveBlogEpicContainer;

for (i = 0; i < liveBlogEpicContainers.length; i++) {
    liveBlogEpicContainer = liveBlogEpicContainers[i];
    liveBlogEpicContainerId = liveBlogEpicContainer.getAttribute('id');
    liveBlogEpicContainer.setAttribute('data-tracked', 'true');
    addEventListenerScroll(liveBlogEpicContainer, liveBlogEpicContainerId);
};

```